### ISSUE TITLE
http.headers.Link - Chromium supports only `preconnect`, `preload` and `modulepreload`

### ISSUE THREAD

**Issue by NiedziolkaMichal** [mdn/browser-compat-data#18816]
### What type of issue is this?

Incorrect support data (ex. Chrome says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

The `Link` header is supposed to be equivalent to the `link` element in HTML, together with all of its attributes. [Chrome 103 has added](https://chromestatus.com/feature/5207422375297024) only `preload`, `preconnect`, and `modulepreload` rel values, while other relations or attributes are still missing.
Firefox correctly recognizes a stylesheet, added by `Link` header value `</path/to/css>; rel="stylesheet"`, so its likely that it supports the whole feature.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `http.headers.Link`
* Report started: 2023-02-03T18:46:35.122Z

</details>

**Comment by caugner** [comment]
It probably makes sense to document this limitation, ideally by adding subfeatures for all possible relations.

However, this would mean researching support for all relations in all browsers, and this isn't really viable to do manually.

If someone wants to manually research support data for `preconnect`, `preload`, `modulepreload` and `stylesheet`, we could start by adding those.

**Comment by caugner** [comment]
This was reported again in https://github.com/mdn/browser-compat-data/issues/29012.

**Comment by tunetheweb** [comment]
> The Link header is supposed to be equivalent to the link element in HTML, together with all of its attributes. [Chrome 103 has added](https://chromestatus.com/feature/5207422375297024) only preload, preconnect, and modulepreload rel values, while other relations or attributes are still missing.

This is incorrect. Chrome usually (always?) added support for them to Link header at the same time as the link elements (e.g. [preload](https://chromestatus.com/feature/5757468554559488).

However, 103 was when 103 early hint support was added. This came much later than regular link header support.

**Comment by caugner** [comment]
> while other relations or attributes are still missing

I think this referred to missing in BCD, not missing in Chrome.

**Comment by caugner** [comment]
Closing as stale.
